drake/examples/Pendulum/runDynamics.cpp

```diff
@@ -0,0 +1,20 @@
+
+#include <iostream>
+#include "Pendulum.h"
+#include "LCMCoordinateFrame.h"
+
+using namespace std;
+
+int main(int argc, char* argv[]) {
+  DrakeSystemPtr p = make_shared<Pendulum>();
```

Comment:
this one is trickier.  right now i have runLCM taking a const shared_ptr &. i don't really want it to, and have decided to correct it (though it's non-trivial)